### DISCUSSION TITLE
Trigger dependency check via Dependabot API in several repos

### DISCUSSION THREAD

**Discussion by ShockwaveNN** [octokit/octokit.rb#1343] [Q&A]
Hello there, I tried to found any documentation for triggering dependabot updates check via api, but seems there is a lot of differences between old dependable and current GitHub owned, so it's not an easy search query to find

In general my user case is rather simple - sometimes I release new version of my dependencies in form of ruby gem and want to all my projects receive this update right now.
Now I just go to ~30 repos and click on Insights tab and check for update manually, but I think there should be better way via api

**Comment by tarebyte** [comment]
@ShockwaveNN at this time there isn't a way to manually trigger a dependabot update check via the GitHub API. I've opened a request ticket up with the owning team and they've added it to their backlog for feature requests.

**Reply by ShockwaveNN** [reply]
Thank you for your response.
Is there a way to track your issue or it's in private repo?

**Reply by tarebyte** [reply]
It's an internal repository 😄
